docs/library/machine.I2CTarget.rst

```diff
@@ -0,0 +1,78 @@
+.. currentmodule:: machine
+.. _machine.I2CTarget:
+
+class I2CTarget -- a device over I2C
+====================================
+
+An I2C target is a device which connects to an I2C bus and is controlled by an
+I2C controller.  I2C targets can take many forms.  The `machine.I2CTarget`
+class implements an I2C target which allows reading and writing to a specific
+part of the target's internal memory.
+
+Example usage::
+
+    from machine import I2CTarget
+
+    # Create the backing memory for the I2C target.
+    mem = bytearray(8)
+
+    # Create an I2C target.  Depending on the port, extra parameters
+    # may be required to select the peripheral and/or pins to use.
+    i2c = I2CTargetMemory(addr=67, mem=mem)
```

Comment:
Shouldn't that be: `i2c = I2CTarget(addr=67, mem=mem)`

Reply:
Now fixed.